src/envoy/http/path_matcher/filter.cc

```diff
@@ -47,6 +51,9 @@ Envoy::Http::FilterHeadersStatus Filter::decodeHeaders(
   } else if (!headers.Path()) {
     rejectRequest(Envoy::Http::Code(400), "No path in request headers.");
     return Envoy::Http::FilterHeadersStatus::StopIteration;
+  } else if (headers.Path()->value().size() > PATH_MAX_SIZE) {
+    rejectRequest(Envoy::Http::Code(400), "Path is too long.");
```

Comment:
also print our limit in the error message.

Reply:
Done